zh/accelerate-v1.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Accelerate 1.0.0"
+thumbnail: /blog/assets/186_accelerate_v1/accelerate_v1_thumbnail.png
+authors:
+- user: muellerzr
+- user: marcsun13
+- user: BenjaminB
+translators:
+- user: hugging-hoi2022
+- user:
```

Comment:
The `proofreader` property can be omitted in your submission, one with `- user: hugging-hoi2022` will be cool!